src/main/java/edu/harvard/iq/dataverse/harvest/client/HarvesterServiceBean.java

```diff
@@ -315,8 +309,7 @@ public void doHarvest(DataverseRequest dataverseRequest, Long harvestingClientId
     
     
     
-    @TransactionAttribute(TransactionAttributeType.NOT_SUPPORTED)
-    public Long processRecord(DataverseRequest dataverseRequest, Logger hdLogger, PrintWriter importCleanupLog, OaiHandler oaiHandler, String identifier, MutableBoolean recordErrorOccurred, MutableLong processedSizeThisBatch, List<String> deletedIdentifiers) {
+    private Long processRecord(DataverseRequest dataverseRequest, Logger hdLogger, PrintWriter importCleanupLog, OaiHandler oaiHandler, String identifier, MutableBoolean recordErrorOccurred, MutableLong processedSizeThisBatch, List<String> deletedIdentifiers, Date dateStamp) {
```

Comment:
I'm curious why "TransactionAttributeType.NOT_SUPPORTED" has been removed. I'm not saying this is a problem, I'm just a little turned around about why this change is being made.